datamade/la-metro-councilmatic
# Conditionally show call in comment instructions

## Overview

This PR hides call in instructions for a subset of committees that do not accept live comment.

### Checklist


## Testing Instructions

 * Scrape agendas for upcoming events:
```bash
docker-compose run --rm scrapers pupa update lametro bills window=7 --rpm=0
docker-compose run --rm scrapers pupa update lametro events window=7 --rpm=0
```
 * View the 3/24 board meeting and confirm you see call in instructions
 * View the 3/31 Measure R meeting and confirm you do not see call in instructions

Handles #820
